### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Guy-Gustave**
What does this PR do?
apple web page
Description of Task to be completed?

The added header of the page
Added nave bar with gradients
added a 
page2
page1

background to the page
added screenshot it README file
page1

How should this be manually tested?
clone the repository
open index.html in your browser
Any background context you want to provide?
N/A
Screenshots (if appropriate)

### CODE REVIEW

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
## Awesome work @Guy-Gustave 
### [The clone](https://raw.githack.com/Guy-Gustave/apple-page/feature-branch/index.html) shows good use of CSS Backgrounds and Gradients. :100: 

A few required changes: Note that for this project, we have to make sure nothing is messed up. 

- [ ] Please see the difference in the source page and the clone below as seen in the desktop screen view 👇  

|[Source Page](https://web.archive.org/web/20140301004610/http://www.apple.com/)|[The Clone](https://raw.githack.com/Guy-Gustave/apple-page/feature-branch/index.html)|
|:---:|:---:|
|source|clone|

- [ ] Validate your markup using this [service tool](https://validator.w3.org/nu/#textarea) to rectify the glaring errors present in the markup.
- [ ] Fix the stickler linter errors.
- [ ] Remove the horizontal scroll in the clone page.
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
- [ ] Check usage of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) elements. They should be "appropriately sized and positioned" (as described in Odin project description) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png) from Pathwright).
- [ ] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright).
##
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by Guy-Gustave — COMMENTED** [review]
after-review
 i submit after correcting all suggested. thanks

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Guy-Gustave, you did a great job fixing some of the comments pointed out by the previous TSE :+1: I would like to highlight the point we missed so we can fix it and get our project approved. Kindly address the comments below: :smile: 
- ```The use of HTML semantics```, let us position the ```<footer>``` after the ```<main>```. Then add ``` margin-left: 5%;``` to the footer
- ```DRY code```, I found ```display:flex``` appears more than 5X. So are the other properties. Let us apply the DRY rule to our code

Kindly make these change and submit another code review request :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by Guy-Gustave** [comment]
hello Daniel, thanks for your suggestions. i changed the <footer>  under
the <main>, but to put margin-left of 5% will change the display of footer
as the original page. better let margin auto. another thing, you tell me
make code dry, I used for 3 times. may I push again if you agree. thanks

**Comment by Guy-Gustave** [comment]
correct the suggested changes,

**Comment by Guy-Gustave** [comment]
correct some suggested changes

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Hello @Guy-Gustave. The changes are coming great with this project. 
### We are almost there.  Just one last chance and we close this.
- [ ] The rendering of the page on a much larger screen is not rendering as it should. This being building with backgrounds, the contents should cover the whole view width irrespective of the screen size.
 Please see the requirements below and counter check with the project's work, thanks.

## Background:
-  The main content should have an image set as background (as CSS [`background-image`](https://www.w3schools.com/cssref/pr_background-image.asp) or [`background -> url`](https://www.w3schools.com/cssref/css3_pr_background.asp)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/background_image_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients)).
- The CSS background should `cover` the whole page irrespective of the screen size. This will be a learning opportunity for the students to use background-size and position properties. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/background_size_requirement.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/)).

- This is the current viewing on a desktop screen view: 👇 Notice the differences with the [source page](https://web.archive.org/web/20140301004610/http://www.apple.com/).
clone
- Kindly work on this last bit and then submit for another code review, Cheers!

Christian Otieno

**Comment by Guy-Gustave** [comment]
hello Chritian, can we meet on zoom?

**Comment by Guy-Gustave** [comment]
hello Christian, I don't know what problem I checked with all browser it
runs well. can we meet on zoom and check together

**Comment by Guy-Gustave** [comment]
see this
Screenshot from 2020-05-28 16-50-25

**Comment by Guy-Gustave** [comment]
now changed background

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+}
+
+.nav-list {
+  display: inline-flex;
+  width: 105px;
+  align-items: center;
+  border-right: 1px solid #606060;
+  height: 35px;
+  justify-content: center;
+  padding-top: 12 px;
```
Comment (rammazzoti2000) [inline css/index.css:39]: `padding-top: 12 px;` wrong syntax, also that rule would add extra padding on the top of the navbar, making it look messy, remove it

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+}
+
+footer a {
+  color: #0ebcf1;
+}
+
+.nav-list a {
+  color: seashell;
+  font-size: 15px;
+  width: 15px;
```
Comment (rammazzoti2000) [inline css/index.css:53]: increase the `width` to `45px` to align all elements on center, the 'Suport' element is not centered

**Review by rammazzoti2000 — APPROVED** [review]
### Hello :wave: @Guy-Gustave I am Alex, your cod reviewer for this project
### Important :exclamation: 
The project overall looks good and is ready for approval except for the search input, it needs some improvements so lets try to make it look better
This is what I have come up with, but feel free to apply your own style:
```
.search {
    position: relative;
    width: 147px;
}

.search img {
    width: 14px;
    color: beige;
    padding-top: 2px;
    position: absolute;
    left: 10px;
}

.nav-list input {
    width: 114px;
    height: 17px;
    background-color: floralwhite;
    border-radius: 6px;
    margin: 0 10px;
    padding-left: 15px;
    outline: none;
}
```
Once you complete the change required, you can then merge this pull request to master branch (remember the changes are require to help student learn more and make the project look professional)
##
<em>Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :heavy_check_mark: Status: Project Approved

File: css/index.css (hunk 2 above)
Comment (Guy-Gustave) [reply css/index.css:53]: thank you. i changed your suggestions

**Pull request merged**